moped-etl/prefect/tasks/activity_log.py

```diff
@@ -0,0 +1,100 @@
+import os
+import json
+import boto3
+
+import prefect
+from prefect import task
+from prefect.tasks.shell import ShellTask
+
+# set up the prefect logging system
+logger = prefect.context.get("logger")
+
+HASURA_ADMIN_SECRET = os.environ["HASURA_ADMIN_SECRET"]
+MOPED_ACTIVITY_LOG_LAMBDA_ROLE_ARN = os.environ["MOPED_ACTIVITY_LOG_LAMBDA_ROLE_ARN"]
+MOPED_ACTIVITY_LOG_QUEUE_URL_PREFIX = os.environ["MOPED_ACTIVITY_LOG_QUEUE_URL_PREFIX"]
+
+# The folder name of the event function and used in naming the AWS function
+function_name = "activity_log"
+
+
+def create_activity_log_aws_name(basename):
+    return f"atd-moped-events-{function_name}_{basename}"
+
+
+def create_activity_log_queue_url(basename):
+    aws_queue_name = create_activity_log_aws_name(basename)
+    return f"{MOPED_ACTIVITY_LOG_QUEUE_URL_PREFIX}/{aws_queue_name}"
+
+
+def create_activity_log_lambda_config(
+    basename,
+):
+    return {
+        "Description": f"AWS Moped Data Event: atd-moped-events-activity_log_{basename}",
+        "Environment": {
+            "Variables": {
+                # We could probably create a helper so this and the ECS Route53 CNAME always match
+                "HASURA_ENDPOINT": f"https://{basename}-graphql.moped-test.austinmobility.io",
```

Comment:
I added this comment so we could align the endpoints generated in the ECS and activity log modules. Not super important.